### DISCUSSION TITLE
Can I configure the file finder to index directories named "build"?

### DISCUSSION THREAD

**Discussion by evankroske** [community/community#23575] [New to GitHub]
<p>Today, the GitHub file finder doesn’t find files in directories named “build”. You can see for yourself by viewing <a href="https://github.com/evankroske/github-doesnt-index-build-dir" rel="nofollow noopener">this repo</a> and pressing “t”. The file finder only finds “notbuild/foo.txt”, not “build/foo.txt”. Is there a way to configure the file finder to index build?</p>

**Comment by natemcmaster** [comment, marked as answer]
<p>I wish there was a way to fix this, too. I reached out to support about this recently and was told the following.</p>
<blockquote>
<p>The file finder (the search you see when using the t keyboard shortcut) does by default exclude these files.</p>
</blockquote>
<blockquote>
<p><strong> It’s not possible to disable or override that in the file finder </strong></p>
</blockquote>
<p>From the github help, it looks like this also applies to other folder names too:</p>
<blockquote>
<p>File finder results exclude some directories like build, log, tmp, and vendor. To search for files within these directories, use the filename code search qualifier.</p>
</blockquote>
<blockquote>
<p><a href="https://help.github.com/articles/finding-files-on-github/" rel="nofollow noopener">https://help.github.com/articles/finding-files-on-github/</a></p>
</blockquote>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/evankroske">@evankroske</a>,</p>
<p>Thanks for the report! Currently, the file finder intentionally excludes directories that commonly contain generated files, including <code>build</code>, <code>log</code>, <code>tmp</code>, and <code>vendor</code>. Omitting what can be a very large list of less-relevant files helps keep the file finder performing well.</p>
<p>That said, we know that these directories can be important in some cases, and we do have an open feature request to allow the exclusions to be configured. Although I can’t say if or when that feature will be added, I’ll add your report to the feature request, and it’ll be evaluated by our product engineering teams.</p>

**Comment by yoannmoinet** [comment]
<p>This is such a weird opinionated and hidden decision.<br>
Not every project have this hierarchy of files, it’s highly dependent on the language too.</p>
<p>Where can we “upvote” this feature request?<br>
It’s been bugging our enterprise account for so long without knowing what was going on.</p>

**Comment by gr8routdoors** [comment]
Idiomatic golang repository structure places [build configuration files into the build directory](https://github.com/golang-standards/project-layout/tree/master/build).  If you're not willing to make this configurable, is there a chance you can have the file finder indexer add `build` for golang repos?
